src/Evolution/Systems/RadiationTransport/M1Grey/Tags.hpp

```diff
@@ -0,0 +1,47 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+#include "Evolution/Systems/RadiationTransport/Tags.hpp"
+
+class DataVector;
+
+/// Namespace for all radiation transport algorithms
+namespace RadiationTransport {
+/// Namespace for the grey-M1 radiation transport scheme
+namespace M1Grey {
+/// %Tags for the evolution of neutrinos using a grey M1 scheme.
+namespace Tags {
+
+/// The densitized energy density of neutrinos of a given species
+/// \f${\tilde E}\f$
+template <typename Fr, class Species>
+struct TildeE : db::SimpleTag {
+  using type = Scalar<DataVector>;
+  static std::string name() noexcept {
+    const Species sp;
+    return Frame::prefix<Fr>() + "TildeE_" + neutrinos::get_name(sp);
```

Comment:
You should just do `Frame::prefix<Fr>() + "TildeE_" + neutrinos::get_name(Species{});` since the variable is only needed for template type deduction :)

Reply:
Thanks - didn't know what the proper syntax was to do this without creating a 'Species' object.

Reply:
You're welcome :) There are lots of these little "tricks" :)